vars/buildDockerAndPublishImage.groovy

```diff
@@ -97,42 +101,45 @@ spec:
         steps {
           container('img') {
             script {
```

Comment:
Unnecessary now?

Reply:
being a script? probably safe to delete now yea